networking/amplify/backend/api/mobile/schema.graphql

```diff
@@ -1,3 +1,25 @@
+type User @model
+@auth(
+  rules:  [
+    { allow: public, provider: iam  },
+    { allow: owner, provider: userPools },
+    { allow: private, operations: [read], provider: userPools },  
+  ]
+)
```

Comment:
I think we'll need to pull the `public` line out of this

Reply:
Or do we? Something to talk about at paired programming I think (I think we already talked about this briefly before)